parser_library/src/workspaces/workspace_configuration.cpp

```diff
@@ -411,7 +411,7 @@ bool workspace_configuration::is_config_file(const utils::resource::resource_loc
 
 bool workspace_configuration::is_b4g_config_file(const utils::resource::resource_location& file) const
 {
-    return file.get_uri().ends_with(B4G_CONF_FILE) && utils::resource::filename(file) == B4G_CONF_FILE;
+    return file.filename() == B4G_CONF_FILE;
```

Comment:
was this causing problems?

Reply:
Not really, no. It was just in a way of writing tests more easily. 

Nevertheless, the `utils::resource::filename(file)` ultimately tries to read the filename directly from a filesystem (via `content_loader.cpp`) and I deemed it unnecessary. Additionally, I found it nicer to unify the behavior with `is_config_file(...)` few lines above where it's checking just the filename without accessing the underlying filesystem.

What do you think?

Reply:
I actually did not realize that `utils::resource::filename(file)` was querying FS, so that definitely was not the right choice :)